sass/_components.scss

```diff
@@ -803,4 +803,173 @@
+            position: relative;
+            transform: rotate($rotation) scale($scaleFactorNarrowColumn);
+
+            &:after {
+                content: "";
+                background: linear-gradient(to top, black 17%, rgba(0, 0, 255, 0) 45%);
+                display: block;
+                position: absolute;
+                top: 0;
+                bottom: 0;
+                left: 0;
+                right: 0;
+                transform: translate(-31%, 4%) rotate(-$rotation) scaleX(2);
+            }
+
+            &.is-7 {
+                transform: rotate($rotation) scale($scaleFactorWideColumn);
+                &:after {
+                    background: linear-gradient(to bottom, black 23%, rgba(0, 0, 255, 0) 77%);
+                    transform: translate(-3%, -13%) rotate(-$rotation);
+                }
+            }
+
+            &:last-child {
+                transform: translateX(40%) rotate($rotation) scale($scaleFactorNarrowColumn);
+            }
+
+            &:hover {
+                img {
+                    opacity: 0;
+                }
+            }
+        }
+
+        img {
+            position: absolute;
+            top: 0;
+            left: 0;
+            transform: translate(-31%, 4%) rotate(-$rotation);
+        }
+
+        .is-2-5 img {
+            height: 100%;
+            max-width: none;
+        }
+            
+        .is-7 img {
+            transform: translate(-3%, -13%) rotate(-$rotation);
+        }
+    }
+
+    .content {
+        color: white;
+        overflow: hidden;
+        height:548px;
+        margin-top: -548px;
+        margin-bottom: 0;
+        position: relative;
+ 
+        .icon-button.fa-play {
+            position: absolute;
+            bottom: $baseline;
+            left: $baseline;
+
+            &:before {
+                bottom: 0;
+                left: 0;
+            }
+        }
+
+        .is-5 {
+            padding: $baseline;
+        }
+
+        .is-3 {
+            padding-bottom: $baseline;
+            padding-left: 1.25rem;
+            padding-right: 1rem;
+            display: flex;
+            flex-direction: column;
+            justify-content: flex-end;
+            
+            h1 {
+                font-size: 2rem;
+            }
+            p {
+                -webkit-line-clamp: 1;
+                margin: 0 0 $baseline/2;
+            }
+        }
+
+        h1 {
+            font-weight: 800;
+            -webkit-line-clamp: 2;
+            -webkit-box-orient: vertical;
+            overflow: hidden;
+            display: -webkit-box;
+        }
+        h2 {
+            text-transform: none;
+            font-weight: 400;
+            font-family: 'Proxima Nova';
+            text-decoration: underline;
+            font-size: 1.5rem
+        }
+        p {
+            -webkit-line-clamp: 3;
+            -webkit-box-orient: vertical;
+            overflow: hidden;
+            display: -webkit-box;
```

Comment:
All these line clamps will require non-webkit code but good enough for the moment